docker-compose.yml

```diff
@@ -8,9 +8,12 @@ services:
       - ../mysql:/var/lib/mysql
       - ../dump:/docker-entrypoint-initdb.d
   web:
-    build: .
+    build: 
+      context: .
+      dockerfile: ${ENV}
```

Comment:
Not sure what this does? How do we point to the right Dockerfile?

Reply:
I have named our Dockerfiles as `development` and `production`, so when we're about to use to docker-compose in production we'll use it as:
`$ ENV=production PORT=3000 docker-compose up` 
and when we're about to use to docker-compose in development we'll use it as:
`$ ENV=development PORT=3000 docker-compose up`